lib/event-source.js

```diff
@@ -0,0 +1,79 @@
+var mongojs = require('mongojs');
+var inflect = require('i')();
+var _ = require('lodash');
```

Comment:
The specs define a "retry" field to be worked by the client, not sure how tinysse's heatbeat helps keeping the connection?